### ISSUE TITLE
Single resource instance nodes in a cardinality n nodegroup

### ISSUE THREAD

**Issue by chiatt** [archesproject/Arches_HER#786]
There are many cases in HER in which cardinality n nodegroups have only on resource instance node. This makes sense if it is likely that every relationship is going to have different relationship properties. However, if the same properties will be used for most relationships, it would make sense to use a resource instance list instead. This would reduce the number of tiles needed to relate instances.

Activity > Associated Consultations
Activity > Associated Digital Files

**Comment by chiatt** [comment]
Here's a query that can be used to identify these cards:
```
select count, nc.nodegroupid, name, graphid from (select count(ng.nodegroupid), ng.nodegroupid from node_groups ng 
inner join nodes n on n.nodegroupid = ng.nodegroupid 
where ng.cardinality = 'n' group by ng.nodegroupid) 
as nc
inner join cards on cards.nodegroupid = nc.nodegroupid
where count = 1
order by graphid, name;
```

**Comment by StucakeGLHER** [comment]
@chiatt @aj-he Hi,  Andy explained this to me, and yes I agree that this would make more sense.

**Comment by Philbox** [comment]
All cardinality changed to 1 except for Images branch where changing cardinality was detrimental

**Comment by aj-he** [comment]
@Philbox - **Activity > Associated Consultations** is still currently set as a `resource-instance` node in a cardinality 1:1 card. Can you fix this to be a `resource-instance-list` as per the others

**Comment by Philbox** [comment]
Fixed. resource instance to resource instance list. Widget changed from select to multiselect

**Comment by chiatt** [comment]
Resolved in #d4df102, #2053ac3
